### DISCUSSION TITLE
Is there anyway to quickly switch back and forth between different adapters?

### DISCUSSION THREAD

**Discussion by k4b7** [remix-run/remix#9086] [Q&A]
I'm using the Fastly adapter for deploys, but it would be nice to use the node adapter for development since it support hot module replacement while the Fastly adapter does not.

**Comment by kiliman** [comment]
I would switch over to Vite. It uses the built-in Remix App Server for development.

https://remix.run/docs/en/main/future/vite#migrating

**Reply by k4b7** [reply]
How does this solve the issue of having to use a different adapter when deploying?
